ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -60,15 +60,35 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: azure-ipi-workers-rhel8-f28
-  cron: 6 9 3 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f14
+  cron: 26 16 6,20 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FILTERS_ADDITIONAL: ~TELCO&
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: azure-ipi-workers-rhel8-f14
```

Comment:
Any background to change frequency to f14 from f28.

Reply:
@shellyyang1989 suggest it because we don't have too many eus upgrade jobs.

Reply:
hi @shellyyang1989 if eus job will be renamed to cpou job, then I think f28 is enough.

Reply:
Hmm, about frequency, I'm thinking if we can balance the test frequency and the test coverage. If we have large amount of jobs, we can decrease the test frequency, otherwise we can increase the test frequency. For the 4.14 eus test, we have only 9 jobs, so i was suggesting running on f14. But I'm also open to your thoughts, if you think f28 makes sense, it's okay for me as well.

Reply:
ok, let's keep f14 for eus upgrade